### DISCUSSION TITLE
[Feature Request] Show time completed and % completed of a book

### DISCUSSION THREAD

**Discussion by ghost** [PaulWoitaschek/Voice#1328] [Ideas]
**Problem:** I can't see how much of the book I've listened to. It only shows the remaining time.

**Suggested solution:** Please show time consumed in this format. <time completed>/<total time> and a % of book completed.

**Current Workaround:** I need to write down the total time in a notes app and then calculate the time completed from the remaining time shown. This is tedious.

% completed is a basic feature on all other audiobook app I've come across so far

**Comment by innesst** [comment]
There's a progress bar beneath the album cover.

**Comment by ghost** [comment]
> There's a progress bar beneath the album cover. 

That looks nice but it doesn't show time completed or % which is helpful as I track my reading/listening progress.

**Comment by ezekiel747** [comment]
this would be nice to have

**Comment by justeepin** [comment]
I logged in to ask for this very thing. 

I love the UI of Voice but prefer the reading stats and tabs in Smart Audiobook. (see screenshot).
![Screenshot_20220904-091344.png](https://user-images.githubusercontent.com/69406371/188315389-20397fb6-0a8e-43cb-b980-5487caba0aba.png)

**Comment by Buckkeep** [comment]
I also came here to request this! I log my reading in an app (TheStoryGraph) and for audiobooks they require either percentage listened or HH:MM listened. Either would be fine for my purposes. I currently work it out like @writerlyhacker mentioned. @justeepin 's solution looks perfect!
Thank you @PaulWoitaschek for this wonderful app! I use it every single day :)
